layers/meta-balena-imx8mplus/recipes-bsp/linux-firmware/linux-firmware_%.bbappend

```diff
@@ -0,0 +1,4 @@
+PACKAGES:remove = " ${PN}-iwlwifi-ty-a0 "
+PACKAGES:remove = " ${PN}-iwlwifi-cc-a0 "
+PACKAGES:remove = " ${PN}-ibt-41-41 "
+PACKAGES:remove = " ${PN}-ibt-20-1-3 "
```

Comment:
Can you remove the space before and after each of these so that we are sure it's not going to run into issues in the future?